Telerik.JustMock/Core/Context/AsyncContextResolver.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ JustMock Lite
+ Copyright © 2019 Progress Software Corporation
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+using System.ComponentModel;
+using System.Reflection;
+
+namespace Telerik.JustMock.Core.Context
+{
+    public static class AsyncContextResolver
+    {
+#if NETCORE
+        static IAsyncContextResolver resolver = new AsyncLocalWrapper();
+#else
+        static IAsyncContextResolver resolver = new CallContextWrapper();
+#endif
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
@lyubomir-rusev Hiding a whole class would be better IMO, it might look a bit strange as it is.